src/olympia/reviewers/templates/reviewers/review.html

```diff
@@ -75,7 +75,7 @@ <h3 id="history">
   <div class="results">
     <div class="results-inner">
       <table id="review-files" class="item-history">
-        {% for version in pager.object_list|reverse %}
+        {% for version in pager.object_list %}
```

Comment:
`|reverse` is no longer needed now that we directly use the queryset, it's already sorted and paginated in SQL.